swig/cpp/src/Struct.h

```diff
@@ -70,19 +70,19 @@ class Val
     Val(sr_val_t *val, S_Counter counter);
     Val(const char *val, sr_type_t type = SR_STRING_T);
     Val(bool bool_val, sr_type_t type = SR_BOOL_T);
-    Val(double decimal64_val, sr_type_t type = SR_DECIMAL64_T);
```

Comment:
@jktjkt what do you want to change in the pull request?

I could change this in the swig file, replace the constructor with the one specified in the swig file or I can add a macro "#ifdef SWIG" in the C++ source code for SWIG specific code.

Those are our two options for now.